hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -2983,5 +2983,121 @@
+        - the test should be run both with / without external clock
+
+      This test should leverage the OTTF test framework for supporting
+      concurrency in a FreeRTOS environment.
+      '''
+      stage: V2
+      tests: []
+    }
+    {
+      name: chip_sw_power_max_load
+      desc: '''Concurrency test modeling maximum load conditions.
+
+      This concurrency test runs multiple blocks at the same time,
+      to simulate maximum load. Should be combined with low power
+      entry and exit scenarios.
+
+      The test should be made configurable so that the type of power state and
+      the time spent in a particular power state can be configured via a
+      flag (or similar). This will make it easier to reuse the test for power
+      simulation and characterization later on.
+
+      Blocks / functionality to run simulatenously in this test:
+
+        - Staggered activation of OTBN, aes, KMAC/HMAC.
+          - KMAC / aes would need to take turns being fed data
+          - KMAC activation should be a combination of otp background, key manager background and software
+          - for OTBN, any signature verification / signing event is sufficient
+        - Entropy complex ongoing
+          - reseed / update operation ongoing
+        - Flash scramble ongoing (ideally both instruction and data, but data should be sufficient for now)
+          - instruction scrambling gated by script availability
+        - Simultaneous IO toggling as defined below
+          - ideally for digital activity, 3xUART / I2C modules should be activated
+            - for first pass simplicity can activate IO portion only for now through GPIO
+          - for dedicated pins, focus on SPI device quad activity
+          - USB activity should be activated
+            - for first pass simplicity activate IO portion only for now via pin forcing in usbdev.
+        - Ongoing cpu activity (icache / SRAM scrambling both activated)
+          - servicing ongoing threads and random read/write data to memory
+          - icache needs to be activated, otherwise the system may spend most of its time fetching code
+        - Background checks enabled wherever possible
+          - rstmgr background checks
+          - alert_handler ping checks
+          - OTP background checks
+
+      This test should leverage the OTTF test framework for supporting
+      concurrency in a FreeRTOS environment.
+      '''
+      stage: V2
+      tests: []
+    }
+    {
+      name: chip_sw_power_idle_load
+      desc: '''Concurrency test modeling load conditions in idle state
+
+      This concurrency test models an average idle scenarios.
+
+      The test should be made configurable so that the type of power state and
+      the time spent in a particular power state can be configured via a
+      flag (or similar). This will make it easier to reuse the test for power
+      simulation and characterization later on.
+
+      The test should cover the following scenarios:
```

Comment:
for both this test and the test below, i would recommend the test itself toggle a GPIO / pin to indicate when the state of interest has begun.  This will make it for Nuvoton to follow along on when exactly should measure.

A log statement in the test would be helpful too.

Reply:
Ack. We may also want to add this to the max load test above.

Reply:
yes good point.

Reply:
Done!